File: worker/src/main/java/space/happyniggersin/worker/WorkerApplication.java
package space.happyniggersin.worker;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.data.mongodb.repository.config.EnableReactiveMongoRepositories;

@SpringBootApplication
@ComponentScan({"space.happyniggersin.common", "space.happyniggersin.worker"})
@EnableReactiveMongoRepositories("space.happyniggersin.common")
public class WorkerApplication {

    public static void main(String[] args) {
        SpringApplication.run(WorkerApplication.class, args);
    }
}

File: worker/src/main/kotlin/space/happyniggersin/worker/WorkerApplication.kt
package space.happyniggersin.worker

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.data.mongodb.repository.config.EnableReactiveMongoRepositories

@SpringBootApplication
@ComponentScan("space.happyniggersin.common", "space.happyniggersin.worker")
@EnableReactiveMongoRepositories("space.happyniggersin.common")
class WorkerApplication

fun main(args: Array<String>) {
    SpringApplication.run(WorkerApplication::class.java, *args)
}
